provision/provision.sh

```diff
@@ -211,6 +214,11 @@ tools_install() {
   npm install -g npm
   npm install -g npm-check-updates
 
+  # nodejs
+  # Install suported version of nodejs
+  curl -sL https://deb.nodesource.com/setup_7.x | bash -
```

Comment:
Note that Shoreditch requires Node 5 to run.  See here: https://github.com/civicrm/civicrm-buildkit/pull/318 where Tim Otten recommends using Node 5 in civicrm-buildkit.

I'm not sure why Shoreditch's build environment is particular, or why civicrm-buildkit is conforming to it, but we may as well keep these in sync.